Makefile

```
@@ -57,118 +68,129 @@ fmt: ## Run go fmt against code.
-
-.PHONY: controller-gen
-controller-gen: $(CONTROLLER_GEN) ## Download controller-gen locally if necessary.
-$(CONTROLLER_GEN): $(LOCALBIN)
-	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_TOOLS_VERSION)
-
-.PHONY: envtest
-envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
-$(ENVTEST): $(LOCALBIN)
-	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
-
-.PHONY: kind
-kind: $(KIND) ## Download kind locally if necessary.
-$(KIND): $(LOCALBIN)
-	GOBIN=$(LOCALBIN) go install sigs.k8s.io/kind@v0.14.0
-
-.PHONY: ginkgo
-ginkgo: $(GINKGO)
-$(GINKGO): $(LOCALBIN) ## Download ginkgo locally if necessary.
-	GOBIN=$(LOCALBIN) go install github.com/onsi/ginkgo/v2/ginkgo@v2.1.4
-
-.PHONY: golangci-lint
-golangci-lint: $(GOLANGCI_LINT)
-$(GOLANGCI_LINT): $(LOCALBIN) ## Download golangci-lint locally if necessary.
-	GOBIN=$(LOCALBIN) go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.46.0
+KUSTOMIZE ?=  $(abspath $(TOOLS_BIN_DIR)/kustomize)
+
+GOLANGCI_LINT := $(abspath $(TOOLS_BIN_DIR)/golangci-lint)
+GINKGO := $(abspath $(TOOLS_BIN_DIR)/ginkgo)
+CONTROLLER_GEN := $(abspath $(TOOLS_BIN_DIR)/controller-gen)
+SETUP_ENVTEST := $(abspath $(TOOLS_BIN_DIR)/setup-envtest)
+GORELEASER := $(abspath $(TOOLS_BIN_DIR)/goreleaser)
+KIND := $(abspath $(TOOLS_BIN_DIR)/kind)
+
+controller-gen: $(CONTROLLER_GEN) ## Build a local copy of controller-gen
+ginkgo: $(GINKGO) ## Build a local copy of ginkgo
+golangci-lint: $(GOLANGCI_LINT) ## Build a local copy of golangci-lint
+setup-envtest: $(SETUP_ENVTEST) ## Build a local copy of envtest
+goreleaser: $(GORELEASER) ## Builds a local copy of goreleaser
+kind: $(KIND) ## Builds a local copy of kind
+kustomize: $(KUSTOMIZE) ## Builds a local copy of kustomize
+
+$(KUSTOMIZE): $(TOOLS_BIN_DIR) ## Build kustomize to the tools binary folder
+	curl -s $(KUSTOMIZE_INSTALL_SCRIPT) | bash -s -- $(subst v,,$(KUSTOMIZE_VERSION)) $(TOOLS_BIN_DIR)
```

This line fails when `make run` is executed the second time.  The rukpak is using `kubectl -k` instead of the standalone kustomize.   Would you make changes not using the standalone kustomize or just add "-" in front of theh`curl` so that the error is ignored?